inference-engine/src/legacy_api/include/legacy/convert_function_to_cnn_network.hpp

```diff
@@ -28,6 +28,8 @@ convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function>& gr
                              CNNNetworkImpl* cnnNetworkImpl,
                              bool keep_constant_inputs = false);
 
+CNNLayer::Ptr createSubGraphLayer(const std::shared_ptr<ngraph::Node>& layer);
```

Comment:
Looks like it is an internal function. Why do we need to define it in the header?

Reply:
Function is defined at the end of file, to use it it is necessary to declare it in the header. I will move function implementation so the declaration is not needed.